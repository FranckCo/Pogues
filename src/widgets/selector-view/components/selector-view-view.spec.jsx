import React from 'react';
import renderer from 'react-test-renderer';

import SelectorViewView from './selector-view-view';

describe('Tab', () => {
  test('Should exists with the corresponding template', () => {
    const props = {
      label: 'Fake label',
      value: 'fake value',
    };
    const selectorViewView = renderer
      .create(
        <SelectorViewView {...props}>
          <div>Fake Children</div>
        </SelectorViewView>,
      )
      .toJSON();

    expect(selectorViewView).toMatchSnapshot();
  });
});
