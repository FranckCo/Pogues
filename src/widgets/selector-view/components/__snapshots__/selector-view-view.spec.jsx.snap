// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tab > Should exists with the corresponding template 1`] = `
<div
  data-label="Fake label"
  data-value="fake value"
>
  <div>
    Fake Children
  </div>
</div>
`;
